### ISSUE TITLE
Salesforce access token requests fail.

### ISSUE THREAD

**Issue by jhorowitz-firedrum** [scribejava/scribejava#823]
Unfortunately, Salesforce is not following the client authentication spec, and only supports client_id and client_secret in the request body.

Commit 82f3833 broke this by not specifying
```
@Override
public ClientAuthenticationType getClientAuthenticationType() {
	return ClientAuthenticationType.REQUEST_BODY;
}
```
in SalesforceApi.java.

**Comment by kullfar** [comment]
Thanks for finding that.
Do you have test app and test user to run the Example, so I can test and merge this fix?

**Comment by jhorowitz-firedrum** [comment]
Sent via email

**Comment by kullfar** [comment]
got test application credentials
need test user credentials as well

**Comment by kullfar** [comment]
Fixed, https://github.com/scribejava/scribejava/commit/eafc4ae2a37ead224aaf86196524786932e321a1 in the master

**Comment by jhorowitz-firedrum** [comment]
Thank you, looking forward to the release to maven central :)

**Comment by kullfar** [comment]
I have just released 5.3.0
It will appear in 15min - 2 hours on the maven central
